cmd/metrics-v2.go

```diff
@@ -79,8 +81,8 @@ func init() {
 		getNetworkMetrics(),
 		getMinioVersionMetrics(),
 		getS3TTFBMetric(),
+		getKMSNodeMetrics(),
```

Comment:
If you add getKMSNodeMetrics() to peerMetricsGroups, then we will able to see the online/offline status of all nodes in /minio/v2/metrics/cluster

```
minio_cluster_kms_online{server="localhost:9001"} 1
minio_cluster_kms_online{server="localhost:9002"} 1
minio_cluster_kms_online{server="localhost:9003"} 1
minio_cluster_kms_online{server="localhost:9004"} 1
```